src/aat/resources/features/F-044/S-579_Later_Case_Read_By_Solicitor_1.td.json

```diff
@@ -0,0 +1,162 @@
+{
+  "title": "Get the case for read by a solicitor",
+  "_extends_": "S-579_Get_Case_For_Case_Worker_Data_Base",
+  "_guid_": "S-579_Later_Case_Read_By_Solicitor_1",
+  "specs": [
+    "to get the same case by Solicitor 1"
+  ],
+  "user": {
+    "username": "befta.caseworker.2.solicitor.1@gmail.com",
+    "password": "Pa55word11"
+  },
+  "request": {
+    "pathVariables": {
+      "cid": "${[scenarioContext][parentContext][childContexts][Befta_Jurisdiction2_Default_Full_Case_Creation_Data][testData][actualResponse][body][id]}",
+      "uid": "[[DYNAMIC]]",
+      "jid": "BEFTA_JURISDICTION_2",
+      "ctid": "BEFTA_CASETYPE_2_1"
+    }
+  },
+  "expectedResponse": {
+    "body": {
+      "id": "[[ANY]]",
+      "jurisdiction": "BEFTA_JURISDICTION_2",
+      "state": "TODO",
+      "version": 1,
+      "case_type_id": "BEFTA_CASETYPE_2_1",
+      "created_date": "[[ANY]]",
+      "last_modified": "[[ANY]]",
+      "security_classification": "PUBLIC",
+      "case_data": {
+        "DocumentField4": {
```

Comment:
Can use inheritance for case_data.

Reply:
no plan for reuse at the moment

Reply:
No, we can't use Standard_Full_Case_Data as the data is different

Reply:
No, we can't use as the data is different